application/F3DStarter.cxx

```diff
@@ -698,6 +701,7 @@ class F3DStarter::F3DInternals
       interactor.addBinding({ mod_t::NONE, "Down" }, "add_current_directories", "Others", std::bind(docString, "Add files from dir of current file"));
       interactor.addBinding({ mod_t::NONE, "F11" }, "take_minimal_screenshot", "Others", std::bind(docString, "Take a minimal screenshot"));
       interactor.addBinding({ mod_t::NONE, "F12" }, "take_screenshot", "Others", std::bind(docString, "Take a screenshot"));
+      interactor.addBinding({ mod_t::CTRL, "O" }, "open_file_dialog", "Others", std::bind(docString, "Open File Dialog"));
```

Comment:
@mwestphal I'm surprised this change do not impact the cheatsheet tests, is it expected?

Reply:
I would definitely expect this change to impact the baselines

Reply:
How does this tests looks like locally @hant-hub ?

Reply:
All the cheatsheet tests pass. I think its because most of them don't have the part of the cheatsheet visible with the changes, and the one which does only has a small difference. So regardless its 90%+ similar to the baseline.

Reply:
Ill check locally

Reply:
So you were almost right. Most of the cheatsheet tests indeed do not show enough of the cheatsheet to be impacted by this change which is quite low in the cheatsheet. The only one that could have been impacted are the `CheatSheetConfigFile` test since the resolution is made to grab the whole cheatsheet, however it so happens that these tests redefine the `Ctrl+O` interaction effectevely making the interaction added here disapear, which is expected. So all good here :)